docs/i18n/en/code.json

```diff
@@ -197,11 +197,11 @@
   },
   "theme.docs.versions.unmaintainedVersionLabel": {
     "message": "This is documentation for {siteTitle} {versionLabel}, which is no longer actively maintained.",
-    "description": "The label used to tell the user that they are browsing an unmaintained doc version"
+    "description": "The label used to tell the user that he's browsing an unmaintained doc version"
```

Comment:
Oh, we can't do this I think.

Reply:
Yeah! We didn't do this Niladri. We never worked on this document & we both (myself & Gincy) are not sure how this got changed.